src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -310,6 +313,15 @@ def makeSubscriptions(self, workflow):
         # add a configurable weight value
         ruleAttrs["weight"] = self.msConfig['rucioDiskRuleWeight']
 
+        # Do not make subscription for relval workflow if not enabled
+        # and consider transfer as done
+        if workflow['IsRelVal'] and self.msConfig['enableRelValDisk']:
+            self.logger.info("Skipping disk subscription for relval workflow",
+                             workflow['RequestName'])
+            workflow.updateTime()
+            self.docKeyUpdate(workflow, TransferStatus='done')
```

Comment:
I think we do not need to have these lines of code over here.
We can likely just let the workflow go through the Pipeline and it should naturally be marked as `done`, of course evaluating whether or not disk data placement needs to be performed or not.

For instance, the code above would likely kick in at:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/MicroService/MSOutput/MSOutput.py#L362